src/frontend/src/pages/AdminToolsPage/AdminToolsFinanceConfig.tsx

```diff
@@ -0,0 +1,148 @@
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableHead,
+  TableRow,
+  TextField,
+  Typography
+} from '@mui/material';
+import PageBlock from '../../layouts/PageBlock';
+import { useGetAllExpenseTypes, useGetAllVendors } from '../../hooks/finance.hooks';
+import LoadingIndicator from '../../components/LoadingIndicator';
+import ErrorPage from '../ErrorPage';
+import { datePipe } from '../../utils/pipes';
+
+const AdminToolsFinanceConfig: React.FC = () => {
+  const { data: vendors, isLoading: vendorIsLoading, isError: vendorIsError, error: vendorError } = useGetAllVendors();
+  const {
+    data: expenseTypes,
+    isLoading: expenseTypesIsLoading,
+    isError: expenseTypesIsError,
+    error: expenseTypesError
+  } = useGetAllExpenseTypes();
+
+  if (!vendors || !expenseTypes || expenseTypesIsLoading || vendorIsLoading) {
+    return <LoadingIndicator />;
+  }
+
+  if (vendorIsError) {
+    return <ErrorPage message={vendorError.message} />;
+  }
+
+  if (expenseTypesIsError) {
+    return <ErrorPage message={expenseTypesError.message} />;
+  }
+
+  const vendorTableRows = vendors.map((vendor) => (
+    <TableRow>
+      <TableCell align="center" sx={{ border: '1px solid black' }}>
+        {datePipe(vendor.dateCreated)}
+      </TableCell>
+      <TableCell sx={{ border: '1px solid black' }}>{vendor.name}</TableCell>
+    </TableRow>
+  ));
+
+  const newVendorRow = (
+    <TableRow>
+      <TableCell sx={{ border: '1px solid black' }} align="center">
+        {datePipe(new Date())}
+      </TableCell>
+      <TableCell sx={{ border: '1px solid black' }}>
+        <TextField placeholder="Enter New Vendor Name" fullWidth size="small"></TextField>
+      </TableCell>
+    </TableRow>
+  );
+
+  const accountCodesTableRows = expenseTypes.map((expenseType) => (
+    <TableRow>
+      <TableCell sx={{ border: '1px solid black' }}>{expenseType.name}</TableCell>
+      <TableCell sx={{ border: '1px solid black' }}>{expenseType.code}</TableCell>
+      <TableCell align="center" sx={{ border: '1px solid black' }}>
+        <Checkbox defaultChecked={expenseType.allowed} />
+      </TableCell>
+    </TableRow>
+  ));
+
+  const registeredVendors = (
+    <Grid item direction="column" textAlign="left" xs={6}>
+      <Typography variant="subtitle1">Registered Vendors</Typography>
+      <TableContainer component={Paper}>
+        <Table>
+          <TableHead>
+            <TableRow>
+              <TableCell
+                align="center"
+                sx={{ fontSize: '16px', fontWeight: 600, border: '1px solid black', width: '25%' }}
+                itemType="date"
+              >
+                Date Registered
+              </TableCell>
+              <TableCell align="left" sx={{ fontSize: '16px', fontWeight: 600, border: '1px solid black' }}>
+                Vendor Name
+              </TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {vendorTableRows}
+            {newVendorRow}
```

Comment:
this is tricky cause we need to be able to create new expense types, new account codes, and be able to edit account codes. I think doing it inline will be pretty tricky, for the mvp, I'd suggest making a modal pop up when you click on a row and then allow the user to make edits in there. 

Same for creating, I think buttons right under both of the tables that pops open a modal for creating a vendor or account code with inputs for all the data a user is inputting will be easier

Lmk if you have questions ab any of that